google-cloud-storage/lib/google/cloud/storage/project.rb

```diff
@@ -299,6 +299,110 @@ def create_bucket bucket_name, acl: nil, default_acl: nil,
           Bucket.from_gapi gapi, service
         end
 
+        ##
+        # Access without authentication can be granted to a File for a specified
+        # period of time. This URL uses a cryptographic signature of your
+        # credentials to access the file identified by `path`. A URL can be
+        # created for paths that do not yet exist. For instance, a URL can be
+        # created to `PUT` file contents to.
+        #
+        # Generating a URL requires service account credentials, either by
+        # connecting with a service account when calling
+        # {Google::Cloud.storage}, or by passing in the service account `issuer`
+        # and `signing_key` values. Although the private key can be passed as a
+        # string for convenience, creating and storing an instance of
+        # `OpenSSL::PKey::RSA` is more efficient when making multiple calls to
+        # `signed_url`.
+        #
+        # A {SignedUrlUnavailable} is raised if the service account credentials
+        # are missing. Service account credentials are acquired by following the
+        # steps in [Service Account Authentication](
+        # https://cloud.google.com/storage/docs/authentication#service_accounts).
+        #
+        # @see https://cloud.google.com/storage/docs/access-control#Signed-URLs
+        #   Access Control Signed URLs guide
+        #
+        # @param [String] bucket Name of the bucket.
+        # @param [String] path Path to of the file in Google Cloud Storage.
```

Comment:
That description was present in both bucket.rb and file.rb. I've corrected all instances of this.